### PR TITLE
fix(platform): menu, fix a11y issues

### PR DESCRIPTION
**Pull request by Betrozov**
## Related Issue

closes https://github.com/sap/fundamental-ngx/issues/6150

## Description

Fixed platform menu a11y issues

##### During Implementation

1. Visual Testing:

-   [n/a] visual misalignments/updates
-   [n/a] check Light/Dark/HCB/HCW themes
-   [n/a] RTL/LTR - proper rendering and labeling
-   [n/a] responsiveness(resize)
-   [n/a] Content Density (Cozy/Compact/(Condensed))
-   [n/a] States - hover/disabled/focused/active/on click/selected/selected hover/press state
-   [n/a] Interaction/Animation - open/close, expand/collapse, add/remove, check/uncheck
-   [n/a] Mouse vs. Keyboard support
-   [n/a] Text Truncation

2. API and functional correctness

-   [n/a] check for console logs (warnings, errors)
-   [n/a] API boundary values
-   [n/a] different combinations of components - free style
-   [n/a] change the API values during testing

3. Documentation and Example validations

-   [n/a] missing API documentation or it is not understandable
-   [n/a] poor examples
-   [n/a] Stackblitz works for all examples

4. Accessibility testing
5. Browser Testing - Edge, Safari, Chrome, Firefox

##### PR Quality

-   [x] the commit message(s) follows the guideline:
        https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
-   [n/a] tests for the changes that have been done
-   [x] all items on the PR Review Checklist are addressed :
        https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
-   [n/a] Run npm run build-pack-library and test in external application
-   [n/a] update `README.md`
-   [n/a] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)

### CODE REVIEW

File: apps/docs/src/app/platform/component-docs/platform-menu/platform-menu-examples/platform-menu-basic-example.component.html [hunk 1]
```diff
@@ -1,19 +1,24 @@
-<fdp-button [fdpMenuTriggerFor]="basicMenu" label="Button" title="Simple Button"></fdp-button>
-<fdp-button buttonType="menu" [fdpMenuTriggerFor]="basicMenu" label="Menu Button" title="Menu Button"></fdp-button>
-<fdp-button glyph="overflow" [fdpMenuTriggerFor]="basicMenu" title="Overflow Icon"></fdp-button>
+<fdp-button [fdpMenuTriggerFor]="basicMenu" label="Button example" [title]="'Simple Button title example'"></fdp-button>
+<fdp-button
+    buttonType="menu"
+    [fdpMenuTriggerFor]="basicMenu"
+    label="Menu Button example"
+    title="Menu Button title"
```
Comment (dimamarksman) [inline apps/docs/src/app/platform/component-docs/platform-menu/platform-menu-examples/platform-menu-basic-example.component.html:6]: shouldn't `[title]` be as binding?

**Review by dimamarksman — APPROVED** [review]

**Review by dmitry-stepanenko — APPROVED** [review]

**Review by platon-rov — APPROVED** [review]

**Comment by Sanjay-1312** [comment]
Hello Colleagues,

Issue 1:
image
OB:  On accessing the 'Button Example' menu button, its UI type conveyed wrongly as 'Button' and Additional information such as 'Tool tip information is also read.
EB: The Button menu UI type should be read and tool tip information should be removed.

For Reference:
https://sapui5untested.int.sap.eu2.hana.ondemand.com/#/entity/sap.m.Menu/sample/sap.m.sample.Menu

Issue 2:
image
OB: On accessing the 'Menu Button Example', menu button, its UI type conveyed wrongly as 'Button' and additional information such as Tool tip information is read.
EB:The correct  UI type Menu button should be conveyed and additional information Tooltip information should be read.

Issue 3:
image

OB: On accessing the 'Grocery menu' button menu, its UI type conveyed wrongly as 'Button' and additional information such tool tio information is read.
EB:The correct  UI type Menu button should be conveyed and additional information Tooltip information should be read.

Issue 4:
image
OB: On accessing the 'Button Example' button menu, its UI type conveyed wrongly as 'Button' and additional information such tool tio information is read.
EB:The correct  UI type Menu button should be conveyed and additional information Tooltip information should be read.


Issue 5:
image
OB: On accessing the 'Button Example' button menu, its UI type conveyed wrongly as 'Button' and additional information such tool tio information is read.
EB:The correct  UI type Menu button should be conveyed and additional information Tooltip information should be read.
This issue persists throughout the Application.

Retest Details:
https://deploy-preview-7269--fundamental-ngx.netlify.app/#/platform/menu
JAWS 2021

**Review by InnaAtanasova — APPROVED** [review]
Btw, you have a change in the library so I updated the title to `fix` :)

**Comment by Sanjay-1312** [comment]
Issue 1: Fixed.

Issue 2: FIxed.

Issue 3: Fixed.

Issue 4: FIxed.

Issue 5: FIxed.

Retest Details:
https://deploy-preview-7269--fundamental-ngx.netlify.app/#/platform/menu
JAWS 2021

**Review by puru-hk — APPROVED** [review]

**Pull request merged**